src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int bottomBase;
+    private int topBase;
+    private int hight;
+
+    public IsoscelesTrapezoid(int bottomBase, int topBase, int hight, Color color) {
+        this.bottomBase = bottomBase;
+        this.topBase = topBase;
+        this.hight = hight;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return ((bottomBase + topBase) / 2) * hight;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("IsoscelesTrapezoid, area: " + getArea() + " sq. units, bottomBase: "
+                + bottomBase + " units, topBase: " + topBase + " units, hight: " + hight

```

Comment:
Ensure that the field name `hight` is corrected to `height` in the `draw()` method as well.